src/server_manager/web_app/digitalocean_account.ts

```diff
@@ -23,12 +23,40 @@ import * as do_install_script from "../install_scripts/do_install_script";
 const SHADOWBOX_TAG = 'shadowbox';
 const MACHINE_SIZE = 's-1vcpu-1gb';
 
+export interface ShadowboxSettings {
+  imageId: string;
+  metricsUrl: string;
+  sentryApiUrl?: string;
+  watchtowerRefreshSeconds?: number;
+}
+
+export enum Status {
+  ACTIVE,
+  EMAIL_UNVERIFIED,
+  INVALID_BILLING_INFORMATION,
+}
+
 export class DigitalOceanAccount implements Account {
   private servers: DigitalOceanServer[] = [];
 
   constructor(
-      private digitalOcean: DigitalOceanSession, private image: string, private metricsUrl: string,
-      private sentryApiUrl: string|undefined, private debugMode: boolean) {}
+      private digitalOcean: DigitalOceanSession, private shadowboxSettings: ShadowboxSettings,
+      private debugMode: boolean) {}
+
+  async getName(): Promise<string> {
+    return (await this.digitalOcean.getAccount())?.email;
+  }
+
+  async getStatus(): Promise<Status> {
```

Comment:
I'm not crazy about creating our own status, but don't have a better idea at the moment. Let me know if you do!

Reply:
I don't see a problem with that. It looks good.